Formula/licensed.rb

```diff
@@ -0,0 +1,47 @@
+class Licensed < Formula
+  desc "Cache and verify the licenses of dependencies"
+  homepage "https://github.com/github/licensed"
+  url "https://github.com/github/licensed.git",
+      tag:      "3.7.3",
+      revision: "76727f75d486a24758890a030e540ebf87bba78b"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  uses_from_macos "ruby"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "licensed.gemspec"
+    system "gem", "install", "licensed-#{version}.gem"
```

Comment:
maybe a nitpick question, can we vendor out these gem resources?

Reply:
@chenrui333 
Thank you for reviewing!

Sorry, I need more detailed explanation because I don't know much about the ruby ecosystem.
Do you mean that I need to install dependencies in the vendor directory?

I am trying with the following code, but it doesn't work at the installation into the libexec directory.
```rb
  def install
    ENV["GEM_HOME"] = libexec
    system "bundle", "config", "--local", "path", "vendor/bundle"
    # This is required to ignore nokogiri error
    system "bundle", "config", "--local", "force_ruby_platform", "true"
    system "bundle", "install"
    system "bundle", "exec", "rake", "install"
    # Now I have the gem file at pkg/licensed-3.7.3.gem
    # However when I try to install this with the following command,
    # gem command starts fetching remote dependencies.
    system "gem", "install", "licensed-#{version}.gem"
    ...
  end
```

Reply:
@chenrui333 
Does "vendor" mean adding all dependencies to the "resource" section like this?
 https://github.com/Homebrew/homebrew-core/blob/master/Formula/asciidoctor.rb#L27

Reply:
Okay, I understand vendor means specifying gems to `resource`s.
https://docs.brew.sh/Formula-Cookbook#specifying-gems-python-modules-go-projects-etc-as-dependencies

Reply:
(BTW, is there any automation tool to vendor out ruby gems?)

Reply:
There is not, the best way is to ensure there is a Gemfile.lock upstream.